tests/preprocessing/casts_int.rs

```diff
@@ -71,7 +89,10 @@ pub const I64_05: i64 = 10_i32;
 pub const I64_06: i64 = 10_u32;
 pub const I64_07: i64 = 10_i64;
 pub const I64_08: i64 = 10_u64;
-pub const I64_09: i64 = 10;
+pub const I64_09: i64 = 10_u128;
+pub const I64_10: i64 = 10_u160;
+pub const I64_11: i64 = 10_u256;
```

Comment:
Is this a legal syntax in Rust?

Reply:
No. This is a shortcut that I have taken for the demo. It saves at least a day of development, if not more. I should have added some comments about this in the K code, I will do that now.